pdpctrl-client/host.go

```diff
@@ -70,197 +74,136 @@ func (h *Host) Close() {
 	h.client = nil
 }
 
-func (h *Host) Process(patch bool, version string, includes map[string][]byte, policy []byte) error {
-	if err := h.upload(patch, version, includes, policy); err != nil {
+func (h *Host) Upload(bucket *DataBucket) error {
+	h.log.Infof("Uploading PDP data to host %s...", h.address)
```

Comment:
We don't need logger any more. Previously error handling was incapsulated to cluster and outer code passes logger to get notion on errors. Now all functions return errors and caller can do whatever it want.